Tasks/JenkinsDownloadArtifacts/ArtifactDetails/ArtifactDetailsDownloader.ts

```diff
@@ -54,11 +49,14 @@ export class ArtifactDetailsDownloader {
             });
         }
         else {
-            let endBuildId: number = parseInt(tl.getInput("jenkinsBuildNumber", true));
-            
-            if (isNaN(endBuildId) || !endBuildId) {
-                defer.reject(new Error(tl.loc("InvalidJenkinsBuildNumber")));
-                return defer.promise;
+            if (jenkinsJobDetails.isMultibranchPipeline) {
+                // if multibranch validate if the branch names are same.
+                let startBuildBranchName: string = this.GetBranchNameFromVersion(startBuildIdStr, jenkinsJobDetails.isMultibranchPipeline);
```

Comment:
knit - use MultiBranchPipeline everywhere (I see it is MutlibranchPipeline right now)

Reply:
done